ang/test/civicase/contact/directives/contact-card.directive.spec.js

```diff
@@ -69,6 +69,20 @@
         });
       });
 
+      describe('when the display name does not have first name or last name but an email id either', function () {
```

Comment:
I don't know if this statement is correct. Is this related to the `display_name` being empty? It seems you wanted to describe something similar to `when the contact doesn't have a a first or last name, but does have a display name`.

Reply:
Fixed